src/scripts/sync_subclassof.py

```diff
@@ -281,6 +281,10 @@ def sync_subclassof(
     logging.info('- Collecting direct subclass relations')
     rels_direct_source_source: Set[RELATIONSHIP] = _get_direct_scr_rels(  # from source
         source_ids, ingest_db, owned_prefix_map)
+    
+    if not rels_direct_source_source or len(rels_direct_source_source) < 1:
+        raise ValueError("Source has no relationships. Something went wrong")
```

Comment:
I add this here because during debugging, I had an issue where one of the sources was not present in the mondo-ingest.db (for some unknown reason), and it would have helped me if they pipeline had failed at this point if no relationships were identified.

There are some corner cases, for example, when zero concepts are mapped to a new source, where this error is false alarm, but I think in the grand scheme of things this indicates a really important issue with one of the inputs.

Reply:
Ash, yes that sounds like a good idea. Maybe I didn't add this initially because the GARD files had no rows. But then when GARD became no longer needed, I didn't think to add this.